### DISCUSSION TITLE
Model type

### DISCUSSION THREAD

**Discussion by elismasilva** [bghira/SimpleTuner#357] [General]
what do you mean by legacy model type? I saw that it has sdxl and legacy, just for clarification, as I would like to know if sd 1.5 could fit into this legacy.

**Comment by bghira** [comment]
legacy includes SD 1.5 but any shape errors will be due to some expectations we get from OpenCLIP.

**Reply by elismasilva** [reply]
i notice you create Dreambooth tutorial, one question, in your example, the text emb  dataset dont need be informed on training data set or in reg dataset?
